install/kubernetes/helm/istio/values.yaml

```diff
@@ -514,7 +514,7 @@ kiali:
   enabled: false
   replicaCount: 1
   hub: docker.io/kiali
-  tag: istio-release-1.0
+  tag: v0.6
```

Comment:
yes but your PR uses v0.6 here, which is not v0.6.0.

Cheers
-steve

Reply:
Yes, but I think v0.6 should include the fix in v0.6.0, @mwringe , can you please confirm?

Reply:
v0.6 represents the latest v0.6.x release. Which right now is v0.6.0.

If we need to push out a fix, we will release v0.6.1 and the v0.6 image will then include any updates in v0.6.1

Reply:
Thanks @mwringe for the clarify, @sdake , can we got this merged?

Reply:
The problem is a zstream release of Kiali may not work with Istio after we release an artifact requiring operators to override the image version back to 0.6.0.

I'm not clear on the semver policies of Kiali.  Does v0.6.1 include feature changes, or only bug fixes?

Cheers
-steve

Reply:
@mwringe can you pleaes show some comments for @sdake 's question?

Reply:
Its meant for critical bug fixes only. Unless its a serious issue users shouldn't be expecting any updates. Users should be expected to pull in a newer version if they want new features and non-critical bugs fixed.